### PR TITLE
[JBPM-7834] Provide OpenShiftStartupStrategy

### PR DESCRIPTION
**Pull request by rhtevan**
Raise the fabric8 OpenShift/K8S client version to 4.1.0 

Associated PR: https://github.com/kiegroup/droolsjbpm-integration/pull/1611

### CODE REVIEW

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by sutaakar** [comment]
jenkins execute full downstream build

**Comment by sutaakar** [comment]
jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -230,7 +230,8 @@
     <version.net.byte-buddy>1.8.17</version.net.byte-buddy>
     <version.docker-client>3.5.12</version.docker-client>
     <version.kubernetes-model>1.1.4</version.kubernetes-model>
```
[the remark this answers is not in the document]
Comment (rhtevan) [reply pom.xml]: The artifact version of kubernetes-model is defined by different property from the overall client. It causes a mis-alignment. Fixed it by using the single property: 'version.kubernetes-client'.

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**